### PR TITLE
Editorial: fix xrefs

### PR DESCRIPTION
**Pull request by marcoscaceres**
***
<a href="https://pr-preview.s3.amazonaws.com/marcoscaceres/geolocation-api/pull/33.html" title="Last updated on Aug 21, 2019, 2:43 PM UTC (e1e5792)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/geolocation-api/33/69a8ee5...marcoscaceres:e1e5792.html" title="Last updated on Aug 21, 2019, 2:43 PM UTC (e1e5792)">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Marked as non substantive for IPR from ash-nazg.

**Comment by marcoscaceres** [comment]
There is a bug in ReSpec that's preventing {{EventListener/handleEvent}} from linking... looking into it.

**Comment by anssiko** [comment]
@marcoscaceres TY! Let us know in this issue if the ReSpec handleEvent bug fix is straight-forward, otherwise I'd be fine to merge this as is and pick the fix later since handleEvent linking was broken in the currently published spec too.

File: index.html [hunk 1]
```diff
@@ -388,7 +388,7 @@ <h3><dfn>Geolocation</dfn> interface</h3>
    return and then asynchronously attempt to obtain the current
    location of the device. If the attempt is successful,
    the <var>successCallback</var> MUST be invoked
-   (i.e. the <code><a data-cite="DOM#dom-eventlistener-handleevent">handleEvent</a></code> operation MUST be called on the
+   (i.e. the {{EventListener/handleEvent}} operation MUST be called on the
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
   (i.e. the {{EventListener/handleEvent()}} operation MUST be called on the
```

File: index.html [hunk 2]
```diff
@@ -434,7 +434,7 @@ <h3><dfn>Geolocation</dfn> interface</h3>
    process</a> and then asynchronously start the watch
    operation. This operation MUST first attempt to obtain the current location of the
    device. If the attempt is successful, the <var>successCallback</var> MUST be invoked
-   (i.e. the <code><a data-cite="DOM#dom-eventlistener-handleevent">handleEvent</a></code> operation MUST be called on the
+   (i.e. the {{EventListener/handleEvent}} operation MUST be called on the
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
   (i.e. the {{EventListener/handleEvent()}} operation MUST be called on the
```

**Comment by marcoscaceres** [comment]
@anssiko, good to merge :)

**Comment by anssiko** [comment]
Thanks @sidvishnoi and @marcoscaceres!

**Pull request merged**